py3votecore/consensus_under_deadline.py

```diff
@@ -0,0 +1,394 @@
+import doctest
```

Comment:
The file should start with a title containing the article name and link,  author names, programmer name, and date.